README.md

```diff
@@ -307,6 +307,15 @@ Each element in the service_providers array must be an object and contain the fo
 ### Processing
 Everything will be validated against the requirements listed above. Once the validation passes, the whole configuration that is in the database is removed and the new configuration is inserted. In other words: the configuration is overwritten.
 
+### LOA Resolution
+It is possible to specify a LOA in 3 places:
+1. The AuthnContextClassRef in the Authentication Request (SAML2 AuthnRequest).
+2. The ```loa``` on the ```service_providers``` configuration.
+3. The ```loa``` on the ```identity_providers``` configuration.
+
+The Gateway will authenticate the user with the highest LOA of all these.
+Second Factor Only mode requires that AuthnRequests use LOA aliases. However these are quickly translated to the equivalent of 'regular' LOAs. The configuration MUST NOT use Second Factor Only LOA aliases.
+
```

Comment:
As for the rest, I'm not particularly familiar with default LOA (only just had them explained to me) and entity specific LOAs. I'll come back to this if there is time at the end.